samples/modules/lvgl/touch_draw/src/main.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2024 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
```

Comment:
Nit: Includes sorting order

Reply:
Yeah there is no guideline per se, but I saw PRs where this order was inforced:
local, zephyr, 3rd party